lib/project_types/script/layers/domain/extension_point.rb

```diff
@@ -26,6 +26,12 @@ def dasherize_type
           @type.gsub("_", "-")
         end
 
+        def library_languages(include_betas: false)
+          @libraries.all.map do |library|
+            include_betas || !library.beta? ? library.language : nil
```

Comment:
I've been thinking this might read nicely if we had a `#stable?` method defined as `!beta?`. Not a big deal though.